### PR TITLE
Aligned to `ckeditor5-dev` ESM changes

### PR DESCRIPTION
**Pull request by psmyrek**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Internal: Aligned to `ckeditor5-dev` ESM changes.

---

### Additional information

This PR requires https://github.com/ckeditor/ckeditor5-dev/pull/1011 to be released. When it is released, all `ckeditor5-dev-*` packages need to be bumped in this repository.

### CODE REVIEW

File: package.json
Comment (pomek) [inline package.json:1]: I am wondering if we could remove the `resolutions` key.

```
  "resolutions": {
    "wrap-ansi": "7.0.0",
    "string-width": "4.1.0",
    "semver": "^7.0.0",
    "ws": "^8"
  },
```

Let's check it once a next major of `ckeditor5-dev` goes live.

**Review by pomek — DISMISSED** [review]

**Review by pomek — APPROVED** [review]

**Comment by pomek** [comment]
@filipsobol, could you help us?

image

**Comment by filipsobol** [comment]
Fixed it by taking the `yarn.lock` from `master` and running `yarn install`. For whatever reason, removing `yarn.lock` and running `yarn install` causes this issue. I will figure out what's causing this issue.

**Review by filipsobol — APPROVED** [review]

**Comment by filipsobol** [comment]
That's likely a bug in a newer version of `vue-tsc` — https://github.com/vuejs/language-tools/issues/4822

**Comment by pomek** [comment]
Thank you!

**Pull request merged**